ocaml/src/albamgr_test.ml

```diff
@@ -20,9 +20,9 @@ open Albamgr_protocol.Protocol
 
 let host = "127.0.0.1"
 let hosts = [ host ]
-let ccfg_ref = ref None
+let ccfg_url_ref = ref None
 
-let get_ccfg () = Option.get_some (!ccfg_ref)
+let get_ccfg_url () = Option.get_some (!ccfg_url_ref)
```

Comment:
Fwiw using Lwt_main.run that is not an insurmountable problem.
E.g. `let ccfg = Lwt_main.run (Alba_arakoon.config_from_url alba_cfg_url) in` should work just fine.